src/components/dialog/dialog.js

```diff
@@ -138,6 +138,9 @@ const VUIDialog = ({
         // Add class to make dialog visible. Needs to occur before focus.
         dialog.classList.add(activeClass);
 
+        // Prevent page scroll
```

Comment:
Nitpicking but really not a fan of "page", I'd go with "viewport" and use "Remove" as that seems to be the more consistent word being used throughout, but that's just me 😄.